root/static/scripts/common/linkedEntities.mjs

```diff
@@ -144,8 +143,14 @@ export function mergeLinkedEntities(
     for (const [type, entities] of Object.entries(update)) {
       if (hasOwnProperty.call(linkedEntities, type)) {
         Object.assign(linkedEntities[type], entities);
-      } else {
+      } else if (type in linkedEntities) {
+        // $FlowIgnore[prop-missing]
         linkedEntities[type] = entities;
+      } else {
+        throw new Error(
+          JSON.stringify(type) +
+          ' is not a valid type assignable to linkedEntities',
+        );
```

Comment:
Did you check that we actually have all the types we're currently using listed under `linkedEntities` now? (not that long ago a few were missing, IIRC)

Reply:
I just checked all the calls to `add_linked_entity` and `link_entity` in Perl and don't see any that aren't listed.

Reply:
Apparently I missed some Entity::Role::Type types, but for some reason the tests didn't fail until https://github.com/metabrainz/musicbrainz-server/pull/2884. I added them in that PR.